packages/container/src/dom/createContainerMessageListener.ts

```diff
@@ -48,7 +49,7 @@ export const createContainerMessageListener: CreateContainerListener = (
     if (isValueChangeMessage(message)) {
       eventHandlers.setContent(message.model)
     } else if (isPluginLoadedMessage(message)) {
-      eventHandlers.setPluginReady()
+      eventHandlers.setPluginReady(message)
```

Comment:
Previously, there was no information in the `loaded` message, but now, we have the `fullHeight` property that needs to be forwarded.